docs/launcher.md

```diff
@@ -1,39 +1,98 @@
+# Multi-Instance vLLM Launcher Documentation
 
-# vLLM Launcher
+## Overview
 
-## Description
+The Multi-Instance vLLM Launcher is a REST API service that allows you to dynamically create, manage, and terminate multiple vLLM inference server instances to achieve model swapping functionality under the Fast Model Actuation (FMA) project. This enables flexible model serving where you can spin up different models on demand, and support multiple concurrent inference workloads.
```

Comment:
> under the Fast Model Actuation (FMA) project

The fact that it is in this repo already says that.

The more interesting thing to say was in the original but this PR currently cuts it out: "achieve model swapping without changes to vLLM". Recall that we have also discussed other ways to achieve model swapping but they involve changing vLLM.